databricks/koalas/generic.py

```diff
@@ -297,12 +297,11 @@ def cumprod(self, skipna: bool = True):
         """
         return self._apply_series_op(lambda kser: kser._cumprod(skipna))  # type: ignore
 
+    # TODO: Although this has removed pandas >= 1.0.0, but we're keeping this as deprecated
     def get_dtype_counts(self):
         """
         Return counts of unique dtypes in this object.
 
-        .. deprecated:: 0.14.0
```

Comment:
We can still deprecate and convert this function to private function only for `DataFrame.info`.  Let's just deprecate and match it to pandas'